util/passenger/passenger.go

```diff
@@ -35,13 +77,15 @@ func (e ErrTokenNotMatchingUser) Error() string {
 	return fmt.Sprintf("passenger: parent of token %+v does not match with actual user %+v", e.Parent, e.Actual)
 }
 
+func init() {
+	passengerKey = key(mathrand.Int63())
```

Comment:
Passenger key is used on lines [293](https://github.com/coduno/api/pull/41/files#diff-15efc5bd6ae02e7b158d8a10f8a35268R293) and [299](https://github.com/coduno/api/pull/41/files#diff-15efc5bd6ae02e7b158d8a10f8a35268R299) as the key for [`context.WithValue](https://godoc.org/golang.org/x/net/context#WithValue).